typing/typecore.ml

```diff
@@ -884,23 +884,36 @@ let constant_or_raise env loc cst =
       c
   | Error err -> raise (Error (loc, env, err))
 
+
+let predef_path_of_optional_module_path = function
```

Comment:
Ah, I see those now. Should really be defined somewhere earlier and used everywhere, not just in typecore.